cmd/hyperkube/kubectl.go

```diff
@@ -19,21 +19,20 @@ package main
 import (
 	"os"
 
-	"k8s.io/kubernetes/cmd/kubectl/app"
+	"k8s.io/kubernetes/pkg/kubectl/cmd"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 )
 
 func NewKubectlServer() *Server {
+	cmd := cmd.NewKubectlCommand(cmdutil.NewFactory(nil), os.Stdin, os.Stdout, os.Stderr)
```

Comment:
I'm okay with skipping `k8s.io/kubernetes/cmd/kubectl` as long as we put a big comment in there that any significant changes need to be duplicated here.  At some point hopefully we can finish the hyperkube transition and hyperkube will be the *only* way to get at this stuff.  Until the, we need to keep stuff in sync.